ndcube/tests/test_ndcubesequence.py

```diff
@@ -134,6 +136,17 @@ def test_cube_like_shape(ndc, expected_shape):
     assert np.all(ndc.cube_like_shape == expected_shape)
 
 
+@pytest.mark.parametrize(("ndc", "expected_dimensions"),
+                         [
+                             ("ndcubesequence_4c_ln_lt_l_cax1", tuple(u.Quantity(d, unit=u.pix) for d in [2. , 12, 4])),
+                         ],
+                         indirect=("ndc",))
+def test_cube_like_dimensions(ndc, expected_dimensions):
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        for ndc_dim, exp_dim in zip(ndc.cube_like_dimensions, expected_dimensions):
+            assert_quantity_allclose(ndc_dim, exp_dim) 
```

Comment:
This looks like a perfectly good solution.  The issue before was that `assert_quantity_allclose` returns `None` if the values are the same, not true.  So an alternative fix would have been

```suggestion
        assert all(assert_quantity_allclose(ndc_dim, exp_dim) is None for ndc_dim, exp_dim in zip(ndc.cube_like_dimensions, expected_dimensions))
```

But if this works, we can keep your solution :)